app/javascript/lib/imperial.test.js

```diff
@@ -2042,680 +2056,705 @@ describe('imperial', () => {
           return game;
         };
 
-        ['maneuver1', 'production1', 'factory', 'taxation'].forEach(
```

Comment:
Unfortunately I had to reduce this to only `maneuver1` case

The new `MoveToRondelSlot` usecase actually enforces taking money from players when moving extra distances, and it'd be tricky to rewrite these to account for that